src/nunit.analyzers.tests/ClassicModelAssertUsage/ClassicModelAssertUsageAnalyzerTests.cs

```diff
@@ -53,148 +51,173 @@ public void VerifySupportedDiagnostics()
         }
 
         [Test]
-        public async Task AnalyzeWhenThatIsUsed()
+        public void AnalyzeWhenThatIsUsed()
         {
-            await TestHelpers.RunAnalysisAsync<ClassicModelAssertUsageAnalyzer>(
-                $"{ClassicModelAssertUsageAnalyzerTests.BasePath}{(nameof(this.AnalyzeWhenThatIsUsed))}.cs",
-                Array.Empty<string>());
+            var testCode = TestUtility.WrapClassInNamespaceAndAddUsing(@"
```

Comment:
Another nitpick: I prefer adding namespace and usings to the string literal over calling this helper. Reason for this I like tests self contained where possible.